train.py

```diff
@@ -2,63 +2,59 @@
-    dev_data = pandora.utils.load_annotated_dir('data/wilhelmus/all_dev',
-                                            format='tab',
-                                            extension='.tab',
-                                            include_pos=params['include_pos'],
-                                            include_lemma=params['include_lemma'],
-                                            include_morph=params['include_morph'],
-                                            nb_instances=None)
-
-    #test_data = pandora.utils.load_annotated_file('data/capitula_classic/test0.tsv',
-    #dev_data = pandora.utils.load_annotated_file('data/mdu/cg-lit/cg-lit_dev.tab',
-    #dev_data = pandora.utils.load_annotated_file('data/EMDu/train.txt',
-    #test_data = pandora.utils.load_annotated_file('data/mdu/cg-lit/cg-lit_test.tab',
-    #                                        format='tab',
-    #                                        include_pos=params['include_pos'],
-    #                                        include_lemma=params['include_lemma'],
-    #                                        include_morph=params['include_morph'],
-    #                                        nb_instances=None)
-    
+    params = pandora.utils.get_param_dict(config)
+    params['config_path'] = config
+    params.update({k: v for k,v in kwargs.items() if v is not None})
+    print("::: Loaded Config :::")
+    for k, v in params.items():
+        print("\t{} : {}".format(k, v))
+
+    train_data = pandora.utils.load_annotated_dir(
+        train,
+        format='tab',
+        extension='.tab',
+        include_pos=params['include_pos'],
+        include_lemma=params['include_lemma'],
+        include_morph=params['include_morph'],
+        nb_instances=None
+    )
+
+    dev_data = pandora.utils.load_annotated_dir(
+        dev,
+        format='tab',
+        extension='.tab',
+        include_pos=params['include_pos'],
+        include_lemma=params['include_lemma'],
+        include_morph=params['include_morph'],
+        nb_instances=None
+    )
     
     tagger = Tagger(**params)
-    tagger.setup_to_train(train_data=train_data,
-                          dev_data=dev_data)
+    tagger.setup_to_train(
+        train_data=train_data,
+        dev_data=dev_data
+    )
     
     for i in range(int(params['nb_epochs'])):
         tagger.epoch()
         tagger.save()
 
     tagger.save()
-    #tagger = Tagger(load=True, model_dir='models/mdu_all')
-
-    
-    
     print('::: ended :::')
 
+
 if __name__ == '__main__':
-    main()
+    parser = argparse.ArgumentParser(description="Training interface of Pandora")
+    parser.add_argument("config", help="Path to retrieve configuration file")
+    parser.add_argument("--dev", help="Path to retrieve configuration file")
+    parser.add_argument("--train", help="Path to retrieve configuration file")
+    parser.add_argument("--nb_epochs", help="Number of epoch", type=int)
```

Comment:
If we we specify this possibility for one hyperparameter, should we not do for every parameter?